CMakeLists.txt

```diff
@@ -63,6 +66,10 @@ if(EPROSIMA_BUILD)
     set(EPROSIMA_BUILD_TESTS ON)
 endif()
 
+if(EPROSIMA_BUILD_TESTS)
+    set(PERFORMANCE_TESTS ON)
```

Comment:
Please, remove these lines according to the last talk.
Could have the agent always the performance test part enabled (without defines)? This code affect the performance? (what would be ironic)

Reply:
Good idea, I removed defined because of the negligible impact in performance.